metricflow-semantics/metricflow_semantics/test_helpers/semantic_manifest_yamls/simple_manifest/semantic_models/user_sm_source.yaml

```diff
@@ -106,3 +106,20 @@ metric:
     measure:
       name: archived_users
       join_to_timespine: true
+---
+metric:
+  name: simple_subdaily_metric_default_day
+  description: simple metric with sub-daily agg time dim that joins to time spine
+  type: simple
+  type_params:
+    measure:
+      name: archived_users
+---
+metric:
+  name: simple_subdaily_metric_default_hour
+  description: simple metric with sub-daily agg time dim that joins to time spine
```

Comment:
nit: these descriptions don't seem right.

Reply:
whoops that was a copy paste for sure.